frontend/src/components/bs-create-space/bs-create-space.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC } from 'react';
+import styles from './bs-create-space.module.scss';
+import { Button } from '../common/button/button';
```

Comment:
```suggestion
import { Button } from 'components/common/common';
```
fix & merge all imports